nuntium/migrations/0057_update_email_confirmation_templates.py

```diff
@@ -0,0 +1,383 @@
+# -*- coding: utf-8 -*-
+import re
+
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+old_default_new_answer_text = u"""Dear {{user}}:
+
+We received an answer from {{person}} to your message "{{message.subject}}" and the answer is:
+
+{{answer.content}}
+
+Thanks for using Write-It
+-- 
+The Write-it Team
+"""
+
+old_default_confirmation_text = u"""Hello {{ confirmation.message.author_name }}:
+We have received a message written by you in {{ confirmation.message.writeitinstance.name }}.
+The message was:
+Subject:  {{ confirmation.message.subject }} 
+Content:  {{ confirmation.message.content|linebreaks }}
+To: {% for person in confirmation.message.people %}
+{{ person.name }}
+{% endfor %}
+
+Please confirm that you have sent this message by copiying the next url in your browser.
```

Comment:
Because somewhere in the past the 'copiying' -> 'copying' typo was fixed this is no longer the default_confirmation_template